internal/db/migrations/002_create_sources_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS sources;
```

Comment:
FYI I propose to not do down migrations in my other PR, so we can just delete this one later on. I don't think it is worth it.